docs/doxygen/overviews/xrc_format.h

```diff
@@ -363,6 +364,19 @@ percent-encoded, e.g. here is the correct way to specify a bitmap with the path
 Bitmap file paths can include environment variables that are expanded if
 wxXRC_USE_ENVVARS was passed to the wxXmlResource constructor.
 
+It is possible to specify the multi-resolution bitmap by a set of bitmaps or SVG file,
+which are mutually exclusive. The set of bitmaps should contain few relative URLs of
+a bitmap, separated by @c ';'.
+Example with a set of bitmaps:
+@code
+<bitmap>new.png;new_2x.png</bitmap>
+@endcode
+While using SVG file you also should specify @c default_size attribute,
+because usually SVG file doesn't have it:
```

Comment:
I'd like to rephrase this as
```suggestion
When using SVG file you must also specify @c default_size attribute
(even if the size is specified in SVG file, it may be different from the size needed here):
```